### PR TITLE
Formatter

### PR DESCRIPTION
**Pull request by DirtyRacer1337**
Add support like this, `{site:|lower} - {date} - {name:|title|replace(" ", '.'))}.{ext:|upper}`

### CODE REVIEW

File: test/namer_types_test.py [hunk 1]
```diff
@@ -92,6 +92,9 @@ def test_formatter(self):
         name = fmt.format("{name}{act: >10}", name="scene1", act="act1")
         self.assertEqual(name, "scene1      act1")
 
+        name = fmt.format("{name:|title}{act:|upper}", name="scene1", act="act1")
```
Comment (4c0d3r) [inline test/namer_types_test.py:95]: Don't understand :|title here?  default value?

Comment (4c0d3r) [reply test/namer_types_test.py:95]: ahhh casing.

**Review by 4c0d3r — APPROVED** [review]
Just would like to understand the use of |:title, but if assuming it's a ninja thing I don't understand.

**Comment by DirtyRacer1337** [comment]
Don't try to find logic, it just because I can

изображение

**Pull request merged**